### PR TITLE
[DO NOT MERGE] Test pull request

### PR DESCRIPTION
**Pull request by khou22**
* This is a test PR for an experimental AI app

### Here is an image

Screenshot 2024-02-12 at 11 28 57 AM

### CODE REVIEW

File: scripts/fib.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+def fib(n: int) -> int:
```
Comment (khou22) [inline scripts/fib.py:1]: Add docstring

```suggestion
def fib(n: int) -> int:
    """
    Calculate the nth Fibonacci number.

    Args:
        n (int): The index of the Fibonacci number to calculate.

    Returns:
        int: The nth Fibonacci number.

    """
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib(n - 1) + fib(n - 2)
```

[Suggestion by Codeium](https://codeium.com//khou22/khou22.github.io/36/review)

File: scripts/fib.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    if n == 0:
+        return 0
+    elif n == 1:
+        return 1
+    else:
+        return fib(n - 1) + fib(n - 2)
```
Comment (khou22) [inline scripts/fib.py:7]: Add docstring

```suggestion
def fib(n: int) -> int:
    """
    Calculate the nth Fibonacci number.

    Args:
        n (int): The index of the Fibonacci number to calculate.

    Returns:
        int: The nth Fibonacci number.

    """
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib(n - 1) + fib(n - 2)
```

[Suggestion by Codeium](https://codeium.com//khou22/khou22.github.io/36/review)

File: scripts/fib.py (hunk 1 above)
Comment (khou22) [inline scripts/fib.py:1]: Add a docstring for this

```suggestion
def fib(n: int) -> int:
    """Calculate the nth Fibonacci number."""
```

[Suggestion by Codeium](https://codeium.com//khou22/khou22.github.io/36/review)

Comment (khou22) [inline scripts/fib.py:1]: Add a docstring to the function

```suggestion
def fib(n: int) -> int:
    """Calculate the Fibonacci number for a given integer n."""
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib(n - 1) + fib(n - 2)
```

[Suggestion by Codeium](https://codeium.com//khou22/khou22.github.io/36/review)

File: scripts/fib.py (hunk 2 above)
Comment (khou22) [inline scripts/fib.py:7]: Add a docstring to the function

```suggestion
def fib(n: int) -> int:
    """Return the nth Fibonacci number."""
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib(n - 1) + fib(n - 2)
```

[Suggestion by Codeium](https://codeium.com//khou22/khou22.github.io/36/review)

Comment (khou22) [inline scripts/fib.py:7]: Add a docstring to the function

```suggestion
def fib(n: int) -> int:
    """Calculate the nth Fibonacci number."""
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib(n - 1) + fib(n - 2)
```

[Suggestion by Codeium](https://codeium.com//khou22/khou22.github.io/36/review)

Comment (khou22) [inline scripts/fib.py:7]: Add docstring

```suggestion
def fib(n: int) -> int:
    """
    Calculate the nth Fibonacci number.
    """
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib(n - 1) + fib(n - 2)
```

[Suggestion by Codeium](https://codeium.com//khou22/khou22.github.io/36/review)

Comment (khou22) [inline scripts/fib.py:7]: Add docstring

```suggestion
def fib(n: int) -> int:
    """
    Calculate the nth Fibonacci number.

    Args:
        n (int): The position of the Fibonacci number to calculate.

    Returns:
        int: The nth Fibonacci number.

    """
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib(n - 1) + fib(n - 2)
```

[Suggestion by Codeium](https://codeium.com//khou22/khou22.github.io/36/review)

File: README.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # Kevin Hou's Personal Website and Blog
 
+This change should not be checked in!
+
```
Comment (mraheja) [inline README.md:4]: testing

```suggestion
The comment "t" does not provide any specific feedback or suggestions for improvement. Please provide more detailed comments or suggestions related to the code change.
```

[Suggestion by Codeium](https://codeium.com//khou22/khou22.github.io/36/review)

File: scripts/fib.py (hunk 2 above)
Comment (mraheja) [inline scripts/fib.py:7]: add docstring

```suggestion
def fib(n: int) -> int:
    """
    Calculates the nth Fibonacci number.
    
    Returns:
        int: The nth Fibonacci number.
    """
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib(n - 1) + fib(n - 2)
```

[Suggestion by Codeium](https://codeium.com//khou22/khou22.github.io/36/review)

File: README.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
...
 [![built with Codeium](https://codeium.com/badges/main)](https://codeium.com?repo_name=khou22%2Fkhou22.github.io) [![streak](https://codeium.com/badges/v2/user/kevin/streak)](https://codeium.com/profile/kevin)
```
Comment (khou22) [inline README.md:5]: Consider adding alt text to the images for accessibility purposes

File: scripts/fib.py (hunk 2 above)
Comment (khou22) [inline scripts/fib.py:7]: Add a docstring to the function

```suggestion
def fib(n: int) -> int:
    """Calculate the nth Fibonacci number using recursive approach. Returns int."""
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib(n - 1) + fib(n - 2)
```

[Suggestion by Codeium](https://codeium.com//khou22/khou22.github.io/36/review)

Comment (khou22) [inline scripts/fib.py:7]: Add a docstring to the function fib

```suggestion
def fib(n: int) -> int:
    """Calculate the nth Fibonacci number."""
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib(n - 1) + fib(n - 2)
```

[Suggestion by Codeium](https://codeium.com//khou22/khou22.github.io/36/review)

Comment (khou22) [inline scripts/fib.py:7]: Add a docstring to the function

```suggestion
def fib(n: int) -> int:
    """Calculate the nth Fibonacci number. Returns 0 if n < 0."""
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib(n - 1) + fib(n - 2)
```

[Suggestion by Codeium](https://codeium.com//khou22/khou22.github.io/36/review)

File: scripts/fib.py (hunk 1 above)
Comment (mraheja) [inline scripts/fib.py:1]: test

**Review by mraheja — COMMENTED** [review]
test

File: scripts/fib.py (hunk 2 above)
Comment (mraheja) [inline scripts/fib.py:7]: add docstring

```suggestion
def fib(n: int) -> int:
    """
    Calculate the Fibonacci number for a given input n.

    Args:
        n (int): The input number for which the Fibonacci number is to be calculated.

    Returns:
        int: The Fibonacci number corresponding to the input n.
    """
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib(n - 1) + fib(n - 2)
```

[Suggestion by Codeium](https://codeium.com//khou22/khou22.github.io/36/review)

File: README.md
Comment (mraheja) [inline README.md:1]: hello

Comment (mraheja) [inline README.md:1]: hi

File: scripts/fib.py
Comment (mraheja) [inline scripts/fib.py:1]: hello

File: README.md
Comment (mraheja) [inline README.md:1]: hi

Comment (mraheja) [inline README.md:1]: test2

Comment (mraheja) [inline README.md:1]: cats

File: README.md [hunk 5]
```diff
@@ -1,5 +1,7 @@
 # Kevin Hou's Personal Website and Blog
 
+This change should not be checked in!
```
Comment (mraheja) [inline README.md:3]: test

**Review by mraheja — CHANGES_REQUESTED** [review]
bruh moment

**Review by mraheja — APPROVED** [review]
bruh moment

**Review by mraheja — APPROVED** [review]
d

File: scripts/fib.py [hunk 6]
```diff
@@ -0,0 +1,10 @@
+def fib(n: int) -> int:
+    if n == 0:
+        return 0
+    elif n == 1:
+        return 1
```
Comment (mraheja) [inline scripts/fib.py:5]: test comment

File: scripts/fib.py [hunk 7]
```diff
@@ -0,0 +1,10 @@
+def fib(n: int) -> int:
+    if n == 0:
+        return 0
```
Comment (mraheja) [inline scripts/fib.py:3]: test

**Review by mraheja — COMMENTED** [review]
d

**Review by mraheja — COMMENTED** [review]
d

**Review by mraheja — COMMENTED** [review]
d

**Review by mraheja — COMMENTED** [review]
l

**Review by mraheja — COMMENTED** [review]
l

**Review by mraheja — APPROVED** [review]
ld

**Review by mraheja — APPROVED** [review]
ld

**Review by mraheja — APPROVED** [review]
good morning

**Review by mraheja — CHANGES_REQUESTED** [review]
good morning

**Review by mraheja — APPROVED** [review]
approved

**Review by mraheja — APPROVED** [review]

**Review by mraheja — APPROVED** [review]
nice

**Review by mraheja — APPROVED** [review]
huge1

File: README.md (hunk 5 above)
Comment (mraheja) [inline README.md:3]: pending1

```suggestion
This change should not be checked in!
```

**Review by mraheja — APPROVED** [review]
d

File: README.md (hunk 5 above)
Comment (mraheja) [inline README.md:3]: pending1

```suggestion
This change should not be checked in!
```

File: scripts/fib.py [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+    elif n == 1:
+        return 1
+    else:
+        return fib(n - 1) + fib(n - 2)
+
+def main() -> None:
+    print(fib(10))
```
Comment (mraheja) [inline scripts/fib.py:10]: testing

```suggestion
def fib(n: int) -> int:
    if n <= 1:
        return n
    return fib(n - 1) + fib(n - 2)

def main() -> None:
    print(fib(10))
```

**Review by mraheja — APPROVED** [review]
something working

File: scripts/fib.py (hunk 2 above)
Comment (mraheja) [inline scripts/fib.py:7]: testing2

```suggestion
        return fib(n - 1) + fib(n - 2)
```

File: README.md (hunk 3 above)
Comment (mraheja) [inline README.md:4]: test3

**Review by mraheja — APPROVED** [review]
testing submit

**Comment by khou22** [comment]
Closing test PR

**Pull request closed without merging**